generator/internal/client_cc_generator.cc

```diff
@@ -70,17 +71,37 @@ bool GenerateClientCC(pb::ServiceDescriptor const* service,
       "gax::StatusOr<$response_object$>\n"
       "$class_name$::$method_name$(\n"
       "$request_object$ const& request) {\n"
-      "  // TODO: actual useful work, e.g. retry, backoff, metadata, "
-      "pagination, etc.\n"
+      "  // TODO: update metadata and idempotency, "
+      "etc.\n"
       "  grpc::ClientContext context;\n"
       "  $response_object$ response;\n"
       "  grpc::Status status = stub_->$method_name$(&context, request, "
```

Comment:
I do not see anything setting a timeout for the request?  If we are not setting the timeout, what stops this request from blocking for approximately forever?

Reply:
This PR is going to be shelved until gax::CallContext is merged. It will be responsible for setting timeout and adding client metadata; this PR will be updated when CallContext has merged.